src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerDisabledIntTest.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.listeners
+
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.matches
+import org.awaitility.kotlin.untilCallTo
+import org.jmock.lib.concurrent.Blitzer
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.springframework.test.context.ActiveProfiles
+import software.amazon.awssdk.services.sns.model.MessageAttributeValue
+import software.amazon.awssdk.services.sns.model.PublishRequest
+import uk.gov.justice.digital.hmpps.personrecord.client.model.hmcts.MessageType.COMMON_PLATFORM_HEARING
+import uk.gov.justice.digital.hmpps.personrecord.integration.MessagingSingleNodeTestBase
+import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
+import uk.gov.justice.digital.hmpps.personrecord.test.messages.commonPlatformHearingWithSameDefendantIdTwice
+import uk.gov.justice.hmpps.sqs.countMessagesOnQueue
+import java.util.UUID.randomUUID
+
+@ActiveProfiles("seeding")
+@Disabled("This works when run in isolation but not as part of check.")
```

Comment:
Is there a not before all to switch application contexts for this test suite? Just a thought

Reply:
I don't know exactly what's going on here. I have an idea that another test is running in parallel with this one when check runs, and that is consuming the message. I will try to verify that by outputting the active profile in the telemetry event but won't be able to commit it.

This change was harder than I thought!

Reply:
I have confirmed that the code consuming the message does not have the profile `seeding` set.
So the code change is working as expected I think.
I am not going to run this today now but I would quite like to get it merged if possible so I can try the seeding tomorrow.